### PR TITLE
Fixed wrong parameters in graphql schema

### PR DESCRIPTION
**Pull request by ChristianDobbie**

### CODE REVIEW

File: routes/spec.graphql [hunk 1]
```diff
@@ -77,12 +77,12 @@ type Query {
     """
     Which stat to sort records by. Requires the full path when used with nested objects like stats.Arcade.wins
     """
-    sort_by: String!
+    sortBy: String!
```
Comment (mattco98) [inline routes/spec.graphql:80]: Seeing as all other parameters (and really all other fields) of the graphql spec are snake case, I'd like to keep it that way for these parameters. To do that, you'd have to change the resolver here: 

https://github.com/slothpixel/core/blob/fa27217a27a622c96846e1657fff8ff5213b0de6/routes/graphql.js#L166-L168

You'd want to accept the snake-case names in the parameter by destructuring them, and pass them into the `leaderboardsAsync` function as camel-case.

Comment (ChristianDobbie) [reply routes/spec.graphql:80]: Are you wanting to dynamically change snake case to camel case or would you prefer mapping each parameter in the graphql spec to their corresponding parameter at the endpoint?

Comment (ChristianDobbie) [reply routes/spec.graphql:80]: As of right now, anything that uses snake case on graphql will not work at all

Comment (mattco98) [reply routes/spec.graphql:80]: They should be changed to snake-case at the endpoint (in the resolver function). Most endpoints destructure the object, pull out the snake-cased name, and pass that directly to a function as a value; those will work fine. Resolver functions that accept the entire object and pass it directly to a function that expects an object with camel-cased names won't work, such as `leaderboard`. They'll have to have their object key names remapped.

**Comment by ChristianDobbie** [comment]
> @ChristianDobbie Would you have interest in continuing with this PR?
Hey sorry, I've been quite busy recently, has this been resolved yet/are you still needing this fixing?

**Comment by builder-247** [comment]
> > @ChristianDobbie Would you have interest in continuing with this PR?
> 
> Hey sorry, I've been quite busy recently, has this been resolved yet/are you still needing this fixing?

The issue hasn't been resolved yet.

**Pull request merged**